src/qibolab/serialize.py

```diff
@@ -22,10 +23,13 @@
 )
 from qibolab.qubits import Qubit, QubitPair
 
+RUNCARD = "parameters.yml"
+PLATFORM = "platform.py"
+
```

Comment:
Here there is any asymmetry between `KERNELS` and `PLATFORM`. The former is defined in kernels.py and imported here while the latter is defined here and important in init.py. Not a big issue, but not sure if it’ll cause any complications.

Reply:
The only problem is that if you do this you generate a circular import.
Currently the `dump` method in `kernels` take a folder and store the kernels in `folder/kernels.npz`, so I would need to import `KERNELS` from `serialize` but `serialize` itself needs `Kernels` to perform the dumping.
A possible workaround would be to let `kernels.dump` accept directly the path to the file and not the folder.
I'm fine with either options.

(Actually a simpler workaround would be just to move kernels which is a single class into `qibolab.serialize` but probably it is a bit ugly)

Reply:
> The only problem is that if you do this you generate a circular import. Currently the `dump` method in `kernels` take a folder and store the kernels in `folder/kernels.npz`, so I would need to import `KERNELS` from `serialize` but `serialize` itself needs `Kernels` to perform the dumping. A possible workaround would be to let `kernels.dump` accept directly the path to the file and not the folder. I'm fine with either options.

Yes, this I expected, the idea was to move `PLATFORM` to init. But as I said not very important, maybe it's fine as it is.

> (Actually a simpler workaround would be just to move kernels which is a single class into `qibolab.serialize` but probably it is a bit ugly)

I agree, this is not a good solution.